### DISCUSSION TITLE
HTTP2 - Chunk upload

### DISCUSSION THREAD

**Discussion by uasan** [uNetworking/uWebSockets.js#345] [Q&A] [answered]
Hi, i know that you do not support chunks in the request body.

But I think many will be interested in learning about the new opportunities web clients, this can be used as a fallback for websockets: [Streaming requests with the fetch API](https://web.dev/fetch-upload-streaming/)

**Comment by ghost** [comment, marked as answer]
If it requires Chrome 85 then it's not a fallback, websockets are supported since Chrome 16 and has 98% global support.

But yes I agree chunked upload should be fixed at some point. Thanks for the post

**Comment by uasan** [comment]
> But yes I agree chunked upload should be fixed at some point. Thanks for the post

Yes, the main thing is to consider all known security vulnerabilities, like:
https://portswigger.net/web-security/request-smuggling
